src/llm_agent.py

```diff
@@ -192,10 +193,12 @@ def init_llm(
         # TODO: support non-openAI llm
         if provider == "openai":
             model_name = model_name or os.getenv("OPENAI_MODEL", "gpt-3.5-turbo")
-
+            client = httpx.Client(proxies={"http://": os.getenv("OPENAI_PROXY"),
+                                           "https://": os.getenv("OPENAI_PROXY")})
```

Comment:
The `httpx.Client` will throw exception when user hasn't define an `OPENAI_PROXY`:

```bash
[2024-02-11, 17:20:49 PST] {subprocess.py:93} INFO - Traceback (most recent call last):
[2024-02-11, 17:20:49 PST] {subprocess.py:93} INFO -   File "/opt/***/run/auto-news/src/af_save.py", line 260, in <module>
[2024-02-11, 17:20:49 PST] {subprocess.py:93} INFO -     run(args)
[2024-02-11, 17:20:49 PST] {subprocess.py:93} INFO -   File "/opt/***/run/auto-news/src/af_save.py", line 232, in run
[2024-02-11, 17:20:49 PST] {subprocess.py:93} INFO -     stat = process_twitter(args)
[2024-02-11, 17:20:49 PST] {subprocess.py:93} INFO -            ^^^^^^^^^^^^^^^^^^^^^
[2024-02-11, 17:20:49 PST] {subprocess.py:93} INFO -   File "/opt/***/run/auto-news/src/af_save.py", line 72, in process_twitter
[2024-02-11, 17:20:49 PST] {subprocess.py:93} INFO -     data_ranked = op.rank(
[2024-02-11, 17:20:49 PST] {subprocess.py:93} INFO -                   ^^^^^^^^
[2024-02-11, 17:20:49 PST] {subprocess.py:93} INFO -   File "/opt/***/run/auto-news/src/ops_twitter.py", line 128, in rank
[2024-02-11, 17:20:49 PST] {subprocess.py:93} INFO -     llm_agent.init_llm()
[2024-02-11, 17:20:49 PST] {subprocess.py:93} INFO -   File "/opt/***/run/auto-news/src/llm_agent.py", line 196, in init_llm
[2024-02-11, 17:20:49 PST] {subprocess.py:93} INFO -     client = httpx.Client(proxies={"http://": os.getenv("OPENAI_PROXY"),
[2024-02-11, 17:20:49 PST] {subprocess.py:93} INFO -              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
[2024-02-11, 17:20:49 PST] {subprocess.py:93} INFO -   File "/home/***/.local/lib/python3.11/site-packages/httpx/_client.py", line 684, in __init__
[2024-02-11, 17:20:49 PST] {subprocess.py:93} INFO -     proxy_map = self._get_proxy_map(proxies or proxy, allow_env_proxies)
[2024-02-11, 17:20:49 PST] {subprocess.py:93} INFO -                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
[2024-02-11, 17:20:49 PST] {subprocess.py:93} INFO -   File "/home/***/.local/lib/python3.11/site-packages/httpx/_client.py", line 225, in _get_proxy_map
[2024-02-11, 17:20:49 PST] {subprocess.py:93} INFO -     proxy = Proxy(url=value) if isinstance(value, (str, URL)) else value
[2024-02-11, 17:20:49 PST] {subprocess.py:93} INFO -             ^^^^^^^^^^^^^^^^
[2024-02-11, 17:20:49 PST] {subprocess.py:93} INFO -   File "/home/***/.local/lib/python3.11/site-packages/httpx/_config.py", line 334, in __init__
[2024-02-11, 17:20:49 PST] {subprocess.py:93} INFO -     raise ValueError(f"Unknown scheme for proxy URL {url!r}")
[2024-02-11, 17:20:49 PST] {subprocess.py:93} INFO - ValueError: Unknown scheme for proxy URL URL('')
[2024-02-11, 17:20:51 PST] {subprocess.py:97} INFO - Command exited with return code 1
[2024-02-11, 17:20:51 PST] {taskinstance.py:1935} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.11/site-packages/airflow/operators/bash.py", line 210, in execute
    raise AirflowException(
```